Documentation/PageTsconfig/TceForm.rst

```diff
@@ -17,53 +17,13 @@ disable options, blind options in selector boxes etc.
 See the core API document section :ref:`FormEngine <t3coreapi:FormEngine>` for more
 details on how records are rendered in the backend.
 
-..  _tceform-apply-properties:
+Read more on how to :ref:`Apply TCEFORM properties <tceform-apply-properties>`.
 
-Applying properties
-===================
+..  toctree::
```

Comment:
I suggest to remove the toctree:

![tceform](https://github.com/TYPO3-Documentation/TYPO3CMS-Reference-TSconfig/assets/2566282/cabe4568-a990-44dd-9f45-1aab3a5c260f)

The applying properties is doubled (sentence above) and the suggest looks displaced. It is linked at the end in the appropriate section. So no need for this toctree IMHO.

Reply:
I made a headline and also listed subtitles as the properties where moved to a subpage. This way still all properties can be seen on this page